src/Network/Network/Generated/LoadBalancer/BackendAddressPool/AddAzureRmLoadBalancerBackendAddressPoolConfigCommand.cs

```diff
@@ -49,6 +49,10 @@ public partial class AddAzureRmLoadBalancerBackendAddressPoolConfigCommand : Net
             HelpMessage = "Name of the backend address pool.")]
         public string Name { get; set; }
 
+        [Parameter(
+            Mandatory = false,
```

Comment:
As your design doc claims, TunnelInterface will be added to Set, Add, New of AzLoadBalancerBackendAddressPool. Why you add it into *-LoadBalancerBackendAddressPoolConfig and update help files of *-AzLoadBalancerBackendAddressPool?

Reply:
actually AzLoadBalancerBackendAddressPool shouldnt be there. we should've named all of them with "Config" because they are not top level resource. Also I fixed the help files so we only have tunnelinterface in *-LoadBalancerBackendAddressPoolConfig